### ISSUE TITLE
Output question regarding good correlation but weird shift in test data.

### ISSUE THREAD

**Issue by JohanRaniseth** [junyanz/pytorch-CycleGAN-and-pix2pix#1168]
I've been using Pix2Pix to generate fake topography images by translating no-snow topography to with-snow topography. Pix2Pix manages to reproduce it well but there is a weird shift. When doing a distribution/histogram plot of a topography image the shapes are more or less the same but there is a shift between them.
Is there anything in Pix2Pix or from your previous experience that explains why Pix2Pix overestimates the values in the fake compared to the ground truth? And possible solutions?

My initial idea was that the distribution/histogram of the training data was different from the test data. For example that the training data was mostly normalized to values between [-0.2,0.2] whereas test was mostly normalized to values between [0.2,0.6]. And that the model then had problems estimating test images that was outside its train dataset range.
But the shift still persists even after removing the shift between training and test data.

Any ideas would be greatly appreciated!

![512_512_compare_dist](https://user-images.githubusercontent.com/31434181/96869167-66968d00-146f-11eb-99f9-fbc90ebbcc8d.png)
![640_512_compare_dist](https://user-images.githubusercontent.com/31434181/96869171-672f2380-146f-11eb-8c8f-558a2a57b2bc.png)
